static/js/custom-render.js

```diff
@@ -21,4 +21,30 @@ Object.assign(render, {
       ? `<img src="${data.location}" style="max-height:150px;" loading="lazy">`
       : null
   },
+  finished: function render(data, type, full, meta, fieldOptions) {
+    if (data == null) return null_column();
+    if (Array.isArray(data) && data.length == 0) return empty_column();
+    data = Array.isArray(data) ? data : [data].map((d) => d === true);
+    if (type != "display") return data.join(",");
+    return `<div class="d-flex">${data
+      .map((d) =>
+        d === true
+          ? `<div class="p-1"><span class="text-center text-info"><i class="fa-solid fa-check fa-lg"></i></span></div>`
+          : `<div class="p-1"><span class="text-center text-secondary"><i class="fa-solid fa-clock fa-lg"></i></span></div>`
+      )
+      .join("")}</div>`;
+  },
+  successful: function render(data, type, full, meta, fieldOptions) {
+    if (data == null) return null_column();
+    if (Array.isArray(data) && data.length == 0) return empty_column();
+    data = Array.isArray(data) ? data : [data].map((d) => d === true);
```

Comment:
```
data = Array.isArray(data) ? data : [data].map((d) => d === true);
```
☝️ I think all this needs to do is ensure data is an Array, i.e...
```
data = Array.isArray(data) ? data : [data]
```
Checking for whether the values `=== true` is happening in the interpolated return value right below, so no need to do it twice.